homeassistant/components/devolo_home_control/strings.json

```diff
@@ -5,18 +5,18 @@
       "already_configured": "This Home Control Central Unit is already in use."
     },
     "error": {
-      "invalid_credentials": "Incorrect user name and/or password."
+      "invalid_credentials": "[%key:common::config_flow::error::invalid_auth%]"
```

Comment:
You could also rename the key to reference key : `invalid_auth` and update the config flow and the tests

Reply:
Done.

Reply:
You need to update the tests as well.